src/wolff_producers.erl

```diff
@@ -248,7 +268,7 @@ handle_info(?refresh_partition_count, #{refresh_tref := Tref, config := Config}
     {noreply, St#{refresh_tref := start_partition_refresh_timer(Config)}};
 handle_info(?rediscover_client, #{client_id := ClientId,
                                   client_pid := false,
-                                  topic := Topic
+                                  topic := TopicOrAlias
```

Comment:
is it really TopicOrAlias here? 
```suggestion
                                  topic := Topic,
                                  alias := Alias
```

Reply:
Yes, `wolff_producers_sup:child_spec` add the alias.